nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-canvas-utils.js

```diff
@@ -516,10 +516,12 @@ nf.CanvasUtils = (function () {
                         });
 
                 // update the tooltip background
-                selection.select('rect.bulletin-background').classed('has-bulletins', true);
+                selection.select('text.bulletin-icon').style("visibility", "visible");
+                selection.select('rect.bulletin-background').style("visibility", "visible").classed('has-bulletins', true);
```

Comment:
With this proposed change we could remove the has-bulletins class and just integrate those styles into the base style of the bulletin-background since we're only ever showing it when it has bulletins.